test/test_confluence.py

```diff
@@ -78,6 +79,7 @@ def test_should_label_url_with_title_in_anchor_for_viewpage() -> None:
     assert_that(label).is_equal_to("SomePage > Säction")
 
 
+# TODO: Add test for Confluence cloud variant - not sure yet how the URL looks like
```

Comment:
I'm not sure what `releaseview.action` is exactly so I couldn't determine the URL for the cloud variant. Maybe we can take a look together?

Reply:
I don't recall. :shrug: May have been a special page type in some Confluence I don't have access to anymore. Unless and until we find an example of that on atlassian.net, we can just leave it out.